Specifications/Events.Store/for_CommittedAggregateEvents/when_creating_with_a_wrong_artifact.cs

```diff
@@ -14,9 +14,9 @@ public class when_creating_with_a_wrong_artifact : given.events_and_an_artifact
     static CommittedAggregateEvents events;
     static Exception exception;
 
-    Establish context = () => wrong_aggregate_event = new CommittedAggregateEvent(wrong_aggregate_artifact, aggregate_version_after + 1, 3, DateTimeOffset.UtcNow, event_source_id, execution_contexts.create(), event_b_artifact, is_public, "wrong");
+    Establish context = () => wrong_aggregate_event = new CommittedAggregateEvent(wrong_aggregate_artifact, aggregate_version_after, 3, DateTimeOffset.UtcNow, event_source_id, execution_contexts.create(), event_b_artifact, is_public, "wrong");
```

Comment:
This change makes me wonder - have we change the meaning of a property on the `CommittedAggregateEvent`? I didn't see a change in that class - so I just want to make sure we have checked all the usages of this property in the codebase :)

Reply:
Yes, I actually added the `AggregateRootVersion` to it. It wasn't there before. We derived the `AggregateRootVersion`  from the last event before.

Reply:
Ohh, this is for one event....

Reply:
In that case the meaning has not changed, so yeah this is a bit weird

Reply:
I've looked at this and the + 1 was just not necessary. This is more correct